qa/logging-config/src/test/java/org/elasticsearch/common/logging/JsonLoggerTests.java

```diff
@@ -111,7 +113,7 @@ public void testDeprecatedMessageWithoutXOpaqueId() throws IOException {
     public void testCompatibleLog() throws Exception {
         withThreadContext(threadContext -> {
             threadContext.putHeader(Task.X_OPAQUE_ID, "someId");
-            final DeprecationLogger testLogger = DeprecationLogger.getLogger("test");
+            final DeprecationLogger testLogger = DeprecationLogger.getLogger("org.elasticsearch.test");
```

Comment:
Why has this changed?

Reply:
the config we used in that test associated `deprecation.test` with deprecation logger. And it is fine for DeprecationLogger in general as there is logic to handle it I am not really sure where we use it though https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/common/logging/DeprecationLogger.java#L64
It was enough for this test to just use DeprecationLogger.getLogger("test").

However ParsedField (and all other usages) are using DeprecationLogger with the class name. So the logger name would be something like `org.elasticsearch....` 
So I thought I will change the name of the deprecation logger in that test.
In fact it is what we have in production log4j2.properties as well.